apps/editorial_features/components/eoy/client.coffee

```diff
@@ -114,16 +129,36 @@ module.exports.EoyView = class EoyView extends Backbone.View
         $('.article-body__intro-header').addClass('active')
     , {offset: '100%'}
 
+  firstSectionInView: =>
+    $('.article-body section[data-section="1"]').waypoint () ->
```

Comment:
#minor: if you're passing an anonymous function without arguments, you don't need to use (). aka, this line could just be:
```
$('.article-body section[data-section="1"]').waypoint ->
```